src/TipBox.vue

```diff
@@ -115,40 +133,68 @@
           default:
             return '<i class="fas fa-exclamation"></i>';
         }
+      },
+      headerContent() {
+        return md.render(this.header);
       }
-    }
+    },
+    mounted() {
+      let id = undefined;
+
+      if (!id && this.headerContent) {
+        const boxHeaderText = jQuery(this.headerContent).wrap('<div></div>').parent().find(':header').text();
+        if (boxHeaderText) {
+            id = slugify(boxHeaderText, { decamelize: false });
+        }
+      }
+
+      if (this.$slots.header) {
+        const header = jQuery(this.$refs.headerContainer.innerHTML).wrap('<div></div>').parent().find(':header');
+        if (header.length > 0) {
+          id = header[0].id;
+        }
+      }
+
+      if (id) {
+        this.$refs.boxContainer.setAttribute('id', id);
+      }
+    },
   }
 </script>
 
 <style scoped>
-    .container {
+    .box-container {
+        width: 100%;
+        padding: 0;
+        border-radius: 6px;
+    }
+
+    .header-wrapper {
         display: flex;
         flex-direction: row;
+        align-items: center;
         width: 100%;
-        border-radius: 6px;
+        padding: 0.28em 1.25rem;
+        border-radius: 6px 6px 0 0;
+    }
+
+    .content-wrapper {
```

Comment:
While this is a class, in MarkBind, we also use this name as an ID to wrap the entire webpage. To avoid confusion, use `tipbox-body` instead?